src/javascripts/ng-admin/Crud/column/maReferencedListColumn.js

```diff
@@ -17,8 +17,9 @@ function maReferencedListColumn(NgAdminConfiguration) {
 <ma-datagrid name="{{ field.datagridName() }}"
     entries="::entries"
     fields="::field.targetFields()"
-    list-actions="::field.listActions()" 
-    entity="::entity">
+    list-actions="::field.listActions()"
+    entity="::entity"
+    datastore="::datastore()">
```

Comment:
Got my response IRL: we can't remove entries, as this is the only way to keep entry ids.